extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -700,6 +705,52 @@ export class SubscriptionService {
     return null;
   }
 
+  /**
+   * Start deferred account creation flow if service config has
+   * appropriate configuration.
+   * @return {?Promise}
+   * @private
+   */
+  maybeStartDeferredAccountFlow_() {
+    return this.platformStore_
+      .getEntitlementPromiseFor('local')
+      .then((localEntitlement) => {
+        if (localEntitlement.grantReason == GrantReason.SUBSCRIBER) {
+          return;
+        }
+        const localServiceConfig = this.platformConfig_['services'].filter(
+          (service) => 'local' === (service.serviceId || 'local')
+        );
+
+        const matchedServiceConfig = userAssert(
```

Comment:
Nit, but alerting here is probably not needed. Let's just include `!matchedServiceConfig` into the `if()` below to exit this flow. From the point of view of this flow: we only run it if local config exists and has the right props.